internal/kibana/client.go

```diff
@@ -34,32 +37,49 @@ type ClientOption func(*Client)
 // NewClient creates a new instance of the client.
 func NewClient(opts ...ClientOption) (*Client, error) {
 	host := os.Getenv(stack.KibanaHostEnv)
-	if host == "" {
-		return nil, stack.UndefinedEnvError(stack.KibanaHostEnv)
-	}
-
 	username := os.Getenv(stack.ElasticsearchUsernameEnv)
 	password := os.Getenv(stack.ElasticsearchPasswordEnv)
+	certificateAuthority := os.Getenv(stack.CACertificateEnv)
 
 	c := &Client{
-		host:     host,
-		username: username,
-		password: password,
+		host:                 host,
+		username:             username,
+		password:             password,
+		certificateAuthority: certificateAuthority,
 	}
 
 	for _, opt := range opts {
 		opt(c)
 	}
+
+	if c.host == "" {
+		return nil, stack.UndefinedEnvError(stack.KibanaHostEnv)
+	}
+
 	return c, nil
 }
 
+// Address option sets the host to use to connect to Kibana.
+func Address(address string) ClientOption {
```

Comment:
I suppose that this function will be used in the original PR, right?

Reply:
Added for completitude, this is only used on tests by now https://github.com/elastic/elastic-package/pull/847/files/bff27e46db7e789f768740b4a4b18cc7c403793b#diff-9c4e0882a886054c20aa4c09e6c0e667826fd54b97f98e5ec36066eb0798feb8R30